streamer/src/nonblocking/quic.rs

```diff
@@ -923,11 +923,17 @@ async fn packet_batch_sender(
                 let len = packet_batch.len();
                 track_streamer_fetch_packet_performance(&packet_perf_measure, &stats);
 
-                if let Err(e) = packet_sender.send(packet_batch) {
+                if let Err(e) = packet_sender.try_send(packet_batch) {
```

Comment:
should we change this to a match? I think these channels are unbounded right now, so this is the only error you could conceivably hit, but probably want a match if we ever bound it.

Reply:
I kind of like how it is now -if Ok we do one thing, if any error we do another. Then match inside checks the specific error - and if disconnected we break.